projects/openmote-b/00std_spi/00std_spi.c

```diff
@@ -56,12 +56,12 @@ void spi_init() {
     // clear variables
     memset(&spi_vars,0,sizeof(spi_vars_t));
 
-    // set the clk miso and cs pins as output
+    // set the clk miso and cs pins as Software-controlled outputs (configures GPIO_AFSEL and GPIO_DIR )
     GPIOPinTypeGPIOOutput(SPI_GPIO_SSI_BASE, SPI_PIN_SSI_CLK);
     GPIOPinTypeGPIOOutput(SPI_GPIO_SSI_BASE, SPI_PIN_SSI_TX);
     GPIOPinTypeGPIOOutput(SPI_GPIO_SSI_BASE, SPI_PIN_SSI_FSS);
 
-    //set cs to high
+    //set cs to high (writes into GPIO_DATA)
     GPIOPinWrite(SPI_GPIO_SSI_BASE, SPI_PIN_SSI_FSS, SPI_PIN_SSI_FSS);
```

Comment:
This is correct, contrary to what I said earlier.
If you look at GPIOPinWrite implementation, it uses address bus filtering (see Figure 9-2 of cc2538 datasheet), so in order to set the ith GPIO pin high, one need to pass 2^i as a data value.
This is exactly how SPI_PIN_SSI_FSS is encoded.